Automattic/wp-calypso
# Remove migration sticker

Related to https://github.com/Automattic/wp-calypso/issues/92032

## Proposed Changes

Removes the migration sticker from the site when the user completes checkout or goes out of the flow. We've covered the usual flows-
- Unmounting/Navigating back from the "Upgrade" page
- Completing the checkout
- Clicking the button for importing the content only

## Why are these changes being made?


* To remove the migration discount sticker from the site

## Testing Instructions



- Enable the feature flag `migration-flow/introductory-offer`. You can add it to your `.env` file like this `ACTIVE_FEATURE_FLAGS="migration-flow/new-migration-instructions-step,migration-flow/introductory-offer"` 
- Start monitoring your browser requests.
- Start a migration and navigate until the step `site-migration-upgrade-plan`.
- Check that when that step loads, it requests the endpoint _.../{SITE_ID}/migration-flow_
- Now click the Back button on top left
- Make sure you see another request sent to the same URL as before, but it's a DELETE request
- Go back to the Migrate/Import page and click on the button for importing content only
- Make sure you see the Delete request being sent
- Now go to the upgrade page, select Upgrade and Migrate, and complete the checkout
- Make sure you see this call being made on the next page

## Pre-merge Checklist
